src/main/java/net/shadowmage/ancientwarfare/automation/tile/warehouse2/TileWarehouse.java

```diff
@@ -34,7 +34,7 @@ public void invalidate() {
 
 	@Override
 	public void handleSlotClick(EntityPlayer player, ItemStack filter, boolean shiftClick, boolean rightClick) {
-		if (!shiftClick && !player.inventory.getItemStack().isEmpty()) {

```

Comment:
This will need to be fixed in a different place below in tryAddItem it call insertItem on every tile in the warehouse multiblock and insert should return back stack that was actually inserted, if every single tile in the warehouse is full that should only return empty stack which doesn't change what player had in their inventory and shouldn't eat any amount of the stack clicked into the warehouse